wasmsdk/blobber.go

```diff
@@ -78,6 +79,15 @@ func getFileStats(allocationID, remotePath string) ([]*sdk.FileStats, error) {
 	return stats, nil
 }
 
+func updateBlobberSettings(blob *sdk.Blobber) (resp string, nonce int64, err error) {
```

Comment:
@boddumanohar  *sdk.Blobber doesn't work in wasm. please use json string instead of it.

Reply:
changed the data input data type to string. Will add the demo too.